src/OpenApi/test/Services/CreateSchemaReferenceIdTests.cs

```diff
@@ -0,0 +1,201 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization.Metadata;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.OpenApi;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+
+public class CreateSchemaReferenceIdTests : OpenApiDocumentServiceTestBase
```

Comment:
Add name collision test?

Reply:
Tq sand your language

Reply:
Added test coverage for this in [`17665da` (#56753)](https://github.com/dotnet/aspnetcore/pull/56753/commits/17665dadbb207ccb10f4bf402cff6fbe8ddd0fc1).